src/bootstrap.php

```diff
@@ -0,0 +1,21 @@
+<?php
+require_once __DIR__.'/../vendor/autoload.php';
+
+ini_set('date.timezone', 'UTC');
```

Comment:
Where is a better location? / What is a better way to set the timezone for the app?